<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera</class>
 <widget class="QWidget" name="camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>683</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>683</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_camera">
      <property name="text">
       <string>打开摄像头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_picture">
      <property name="text">
       <string>拍照</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_return1">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>选择设备</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
